test/Engine/ProtoTest/Associative/MethodResolution.cs

```diff
@@ -388,6 +388,30 @@ public void MAGN5729_Repro_Simple()
             thisTest.Verify("r3", new object[] { true, false });
         }
 
+        [Test]

```

Comment:
The test should use an argument having the same name as the function in order to test this fix, right?